src/widgets/refinement-list/__tests__/refinement-list-test.js

```diff
@@ -187,17 +179,6 @@ describe('refinementList()', () => {
       createURL = () => '#';
     });
 
-    it('formats counts', () => {
```

Comment:
This is not the place to test this, applied helpers are already tested in Template-test